examples/multiedgetracking/multi-edge-tracking-service-kafka.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: sedna.io/v1alpha1
```

Comment:
I suggest that a proposal for ReID service feature like [this](https://github.com/kubeedge/sedna/blob/main/docs/proposals/object-tracking.md) should be submit, and there should be some description of solution.

Reply:
I will extend this PR with some extra details.

Reply:
What's the relationship between multiedgetracking and the existing objecttracking?

Reply:
> What's the relationship between multiedgetracking and the existing objecttracking?

I'm confused, didn't we discuss this point in the last two weeks' meetings? This is our ReID solution and the code was asked to be released to merge it into Sedna.